theme-base/README.md

```
@@ -0,0 +1,32 @@
+
+The build script (`scripts/build-themes.js`) combines:
+1. Base token colors from `gruvbox-crisp-base.json`
+2. Base UI colors from `ui-colors-base.json`
+3. Variant-specific background colors from `variants.json`
+
+This generates the complete theme files in the `themes/` directory.
+
+## Building Themes
+
+Run the build command from the project root:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)